release_notes/unreleased.md

```diff
@@ -1 +1,2 @@
 **Unreleased**
+* Updated dependencies to resolve security vulnerabilities [PSAAS-20398]
```

Comment:
To be more precise you've removed them from the package so the platform ones are now used.